wire-moshi-support/src/test/java/com/squareup/wire/MoshiTest.java

```diff
@@ -0,0 +1,298 @@
+/*
+ * Copyright 2013 Square Inc.
```

Comment:
nit: 2018?

Reply:
I remember reading somewhere that this should be aligned on all files. That new files should keep the date for the first copyrighted file.

Reply:
I was wrong. Fixed it.